### PR TITLE
Reorganize template brand update

### PR DESCRIPTION
**Pull request by jomaora**
**Detailed purpose of the PR**

In order to have the same template with left navigation and header for setup, this PR standardises the distribution of components in the Brand Update Template.

**Result and observation**

## Setup Header
The header was updated. Currently in prod, it does not resizes it self in desktop width, and the `Platform List` button takes a lot of space, and it was breaking the new disposition of the template.

#### Now
Capture d’écran 2021-11-23 à 16 32 19
#### After
Capture d’écran 2021-11-23 à 16 32 23

## Template Brand Update

There was a border wrapping all the forms and existing views from setup. This border does not exist anymore. And as you can see, the padding was generalized for all fixtures, no matter with kind of content type we pass to the template (`form, home, analytics-dashboards, list`)

### Now 
Kapture 2021-11-23 at 16 44 18

### After
Kapture 2021-11-23 at 16 49 21

## Notifications

The new Banner molecule was added to replace the older notifications

### Now
Capture d’écran 2021-11-23 à 16 53 42

### After
Capture d’écran 2021-11-23 à 16 53 38

### CODE REVIEW

File: packages/@coorpacademy-components/src/template/back-office/brand-update/index.js [hunk 1]
```diff
@@ -50,90 +49,108 @@ const BrandUpdate = props => {
...
+    filter(e => e.selected),
+    head
+  )(items);
+
+  return selectedTab && !isEmpty(selectedTab.subTabs) ? (
+    <div>
+      <BrandTabs type="light" tabs={selectedTab.subTabs} />
+    </div>
+  ) : null;
+};
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/index.js:110]: In these cases it is way better to have early returns, makes it more readable
```suggestion
  const showTabs = selectedTab && !isEmpty(selectedTab.subTabs);
  if (!showTabs) return null;
  return (
    <div>
      <BrandTabs type="light" tabs={selectedTab.subTabs} />
    </div>
  );
};
```

File: packages/@coorpacademy-components/src/template/back-office/brand-update/style.css [hunk 2]
```diff
@@ -13,14 +13,21 @@
...
 }
 
+.header {
+  margin: 24px 0px 56px 0px;
+}
+
+.headerForNotifications {
+  margin: 24px 0px 0px 0px;
+}
+
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/style.css:30]:
```suggestion
.header {
  margin: 24px 0 56px 0;
}

.headerForNotifications {
  margin: 24px 0 0 0;
}

```

File: packages/@coorpacademy-components/src/template/back-office/brand-update/style.css [hunk 3]
```diff
@@ -30,15 +37,8 @@
...
-.contentWrapper {
-  width: 100%;
-  display: flex;
-  flex-direction: column;
-  overflow: auto;
-}
-
 .notifications {
-  padding: 35px 120px 0;
+  padding: 24px 0px 32px 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/style.css:41]:
```suggestion
  padding: 24px 0 32px 0;
```

File: packages/@coorpacademy-components/src/template/back-office/brand-update/style.css [hunk 4]
```diff
@@ -49,35 +49,18 @@
...
-  padding: 1rem;
-  text-align: left;
-  position: relative;
-  margin: 0 auto;
-  margin-top: 16px;
-  line-height: 1.4em;
-  border: 1px solid light;
+.contentWrapper {
+  width: 100%;
+  padding: 0px 64px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/style.css:60]:
```suggestion
  padding: 0 64px;
```

Eslint warns about this

File: packages/@coorpacademy-components/src/template/back-office/brand-update/style.css [hunk 5]
```diff
@@ -103,46 +86,15 @@
...
-
 @media tablet {
   .container {
     scroll-behavior: unset;
-  }
-
-  .contentView {
-    margin: 0 auto;
-    margin-right: 16px;
+    margin: 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/style.css:92]:
```suggestion
    margin: 0;
```

File: packages/@coorpacademy-components/src/organism/setup-header/style.css [hunk 6]
```diff
@@ -147,6 +147,20 @@
   display: none;
 }
 
+@media desktop {
+  .platformListIcon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 16px 0px 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/organism/setup-header/style.css:155]:
```suggestion
    padding: 0 16px 0 0;
```

File: packages/@coorpacademy-components/src/molecule/brand-form-group/style.css [hunk 7]
```diff
@@ -11,6 +11,11 @@
   margin-bottom: 20px;
 }
 
+.titleWrapper:first-child {
+  margin-top: 0px;
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/molecule/brand-form-group/style.css:15]:
```suggestion
  margin-top: 0;
```

File: packages/@coorpacademy-components/src/template/back-office/brand-update/index.js [hunk 8]
```diff
@@ -13,33 +13,32 @@ import BrandTable from '../../../organism/brand-table';
...
+          target={'_blank'}
+        />
+      ) : (
+        <LinkItem {...subTab} styles={getStyle(subTab.selected)} uppercase={false} />
+      )}
+    </div>
+  ))(_subTabs);
 
+// TODO: this fonction should be replaced by a molecule to avoid this file to be unreadable
+const buildLeftNavigation = (logo, items, onItemClick) => {
```
Comment (Fausto95) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/index.js:41]: I'd leave it coupled here, makes the look up easier..

**Review by Fausto95 — CHANGES_REQUESTED** [review]

**Review by Fausto95 — APPROVED** [review]

**Review by StefanoA1 — APPROVED** [review]

File: packages/@coorpacademy-components/src/template/back-office/brand-update/index.js [hunk 9]
```diff
@@ -50,90 +49,117 @@ const BrandUpdate = props => {
...
+};
 
+const buildTabs = items => {
+  const selectedTab = pipe(
+    filter(e => e.selected),
+    head,
+    get('tabs'),
+    filter(e => e.selected),
+    head
+  )(items);
```
Comment (silouone) [inline packages/@coorpacademy-components/src/template/back-office/brand-update/index.js:103]: filter => head c'est un find non ? 


```suggestion
 const selectedTab = pipe(
    find(e => e.selected),
    get('tabs'),
    find(e => e.selected)
  )(items);
```

Comment (silouone) [reply packages/@coorpacademy-components/src/template/back-office/brand-update/index.js:103]:
```suggestion
    const selectedTab = pipe(find('selected'), get('tabs'), find('selected'))(items);

```

**Review by silouone — APPROVED** [review]

**Review by DjamelSoualmi — APPROVED** [review]

**Pull request merged**